io/js/src/main/scala/fs2/io/process/ProcessesPlatform.scala

```diff
@@ -36,19 +36,47 @@ private[process] trait ProcessesCompanionPlatform {
     def spawn(process: ProcessBuilder): Resource[F, Process[F]] =
       Resource {
         F.async_[(Process[F], F[Unit])] { cb =>
-          val childProcess = facade.child_process.spawn(
-            process.command,
-            process.args.toJSArray,
-            new facade.child_process.SpawnOptions {
-              cwd = process.workingDirectory.fold[js.UndefOr[String]](js.undefined)(_.toString)
-              env =
+          
+
+          val spawnOptions = js.Dynamic.literal {
```

Comment:
Instead of using `js.Dynamic` can you update the `SpawnOptions` face to add the additional fields?